lib/web_ui/test/html/paragraph/text_goldens.dart

```diff
@@ -13,29 +13,28 @@ import 'package:web_engine_tester/golden_tester.dart';
 import 'helper.dart';
 
 /// Class that controls some details of how screenshotting is made.
-///
-/// (For Googlers: Not really related with internal Scuba anymore)
-class EngineScubaTester {
-  EngineScubaTester(this.viewportSize);
+class EngineGoldenTester {
+  EngineGoldenTester(this.viewportSize);
 
-  /// The size of the browser window used in this scuba test.
+  /// The size of the browser window used in this golden test.
   final ui.Size viewportSize;
 
-  static Future<EngineScubaTester> initialize(
+  static Future<EngineGoldenTester> initialize(
       {ui.Size viewportSize = const ui.Size(2400, 1800)}) async {
     assert(() {
       if (viewportSize.width.ceil() != viewportSize.width ||
           viewportSize.height.ceil() != viewportSize.height) {
         throw Exception(
-            'Scuba only supports integer screen sizes, but found: $viewportSize');
+            'Gold only supports integer screen sizes, but found: $viewportSize');
       }
       if (viewportSize.width < 472) {
-        throw Exception('Scuba does not support screen width smaller than 472');
+        // TODO: Do we still need this check?
```

Comment:
@ditman do you know if this is still necessary?

Reply:
That comment must be from 2019, I don't think we have any limitations of Scuba any longer!

Reply:
(I cannot find anything anywhere mentioning 472 pixels anymore, and I don't remember that :P)

Reply:
Thanks for checking! I'll remove it then.